extensions/wikia/CommunityPage/styles/entrypoint/EntryPoint.scss

```diff
@@ -1,61 +1,54 @@
 @import 'skins/shared/color';
 @import 'skins/oasis/css/core/breakpoints-variables';
 @import '../components/variables';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';
 
-$background-color-dark: #1a1a1a;
-$background-color-light: #f6f6f6;
+$logo-url: '/extensions/wikia/DesignSystem/bower_components/design-system/dist/svg/wds-company-logo-fandom-heart.svg';
 
-$background-color: $background-color-light;
-@if $is-dark-wiki {
-	$background-color: $background-color-dark;
-}
-
-$logo-url-dark: '/extensions/wikia/CommunityPage/images/wikia-logo-dark.svg';
-$logo-url-light: '/extensions/wikia/CommunityPage/images/wikia-logo.svg';
-
-$logo-url: $logo-url-light;
-@if $is-dark-wiki {
-	$logo-url: $logo-url-dark;
-}
+$button-height: 46px;
+$logo-size: 30px;
+$logo-border-radius: $logo-size / 2;
 
 .WikiaRail {
 	.module {
 		position: relative;
 
 		&.community-page-entry-point-module {
-			padding-top: 40px;
+			padding-top: $content-spacing-xxlarge;
 		}
 	}
 }
 
 .community-page-entry-point-module {
 	font-family: $community-page-font-family;
-	font-size: 16px;
-	line-height: 21px;
-	margin-top: 20px;
+	font-size: $typescale-size-base;
+	line-height: $module-default-line-height;
+	margin-top: $content-spacing-regular;
```

Comment:
I compared how it look after this difference is minor, so I decided to use existing spacing value